ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/text/v1/text/text.html

```diff
@@ -18,7 +18,6 @@
      data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
      data-cmp-is="adaptiveFormText"
      id="${textModel.id}"
-     name="${textModel.name}"
```

Comment:
Why do you need to remove name here ?

Reply:
name attribute cannot be used on div, it is reported by customer for accessibility enhancement